suite/tests/security-common/selfmod.c

```diff
@@ -544,6 +559,27 @@ ADDRTAKEN_LABEL(loop_target_end3:)
 #undef FUNCNAME
 
 
+#define FUNCNAME sandbox_direction_flag
+        DECLARE_FUNC(FUNCNAME)
+GLOBAL_LABEL(FUNCNAME:)
+DECLARE_GLOBAL(sandbox_direction_flag_no_ilt)
+ADDRTAKEN_LABEL(sandbox_direction_flag_no_ilt:)
+        mov      REG_XAX, HEX(1)
+        lea      REG_XDX, SYMREF(direction_flag_immediate_addr_plus_four - 4)
+        mov      DWORD [REG_XDX], eax        /* selfmod write */
+        mov      REG_XDX, HEX(0)             /* mov_imm to modify */
+ADDRTAKEN_LABEL(direction_flag_immediate_addr_plus_four:)
```

Comment:
it would be helpful to add comment to explain the name, i.e., why _plus_four.
Does it work for both x86 and x86_64, maybe show the real instruction encoding.